src/Card/_variables.scss

```diff
@@ -41,7 +41,7 @@ $card-logo-bottom-offset-horizontal: .4375rem !default;
 $card-logo-width:                    7.25rem !default;
 $card-logo-height:                   4.125rem !default;
 
-$card-image-border-radius:           .3125rem !default;
+$card-image-border-radius:           $card-border-radius !default;
```

Comment:
@adamstankiewicz was the intention of [this comment](https://github.com/openedx/paragon/issues/2498#issuecomment-1665549526) to handle things this way, or to just use `$card-border-radius` everywhere directly?

Reply:
In [this](https://github.com/openedx/paragon/pull/2496) PR @adamstankiewicz proposed to unify specific border-radius variables for Card and CardImg.

Reply:
@PKulkoRaccoonGang @brian-smith-tcril I believe it would be preferable to have a single `$card-border-radius` used everywhere directly. That said, if we remove `$card-image-border-radius`, any brand authors that might have overridden that variable to customize the brand would no longer be able to. That said, it's probably unlikely many brand authors are modifying this variable 🤷‍♂️ 

Perhaps we plan to keep it with `$card-image-border-radius: $card-border-radius` as currently implemented for now and then remove it fully (in favor of using `$card-border-radius` everywhere directly) in the `alpha` breaking release?